### PR TITLE
Set up database connection

### PR DESCRIPTION
**Pull request by nikzaugg**
## Database Connection

**Issue:** #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# poster-service
\ No newline at end of file
+# poster-service
+
+## Database
+
+**Postgres:** We use the free postgres databse offered by heroku
```
Comment (rschlaefli) [inline README.md]:
```suggestion
**Postgres:** We use the free postgres database addon offered by Heroku
```

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+
+- https://devcenter.heroku.com/articles/heroku-postgresql#connecting-in-node-js
+- https://devcenter.heroku.com/articles/connecting-to-heroku-postgres-databases-from-outside-of-heroku#credentials
+
+## What to do
+
+- create an `.evn` file with the `.env.template`
```
Comment (rschlaefli) [inline README.md]:
```suggestion
- create a `.env` file based on `.env.template`
```

File: README.md (hunk 1 above)
Comment (nikzaugg) [reply README.md]: resolved in ebb3c42

File: README.md (hunk 2 above)
Comment (nikzaugg) [reply README.md]: resolved in ebb3c42

**Pull request merged**